src/main/java/org/isf/utils/jobjects/MessageDialog.java

```diff
@@ -32,7 +32,6 @@
 	private static final String ERROR_MESSAGE = MessageBundle.getMessage("angal.common.error.title");
 	private static final String WARNING_MESSAGE = MessageBundle.getMessage("angal.common.warning.title");
 	private static final String INFO_MESSAGE = MessageBundle.getMessage("angal.common.info.title");
-	private static final String QUESTION_MESSAGE = MessageBundle.getMessage("angal.common.question.title");
```

Comment:
no longer needed?

Reply:
It will come back soon; the implementation that I had there was incorrect and didn't function the way I wanted.   Until I have time to circle back to it I just removed the constant and the method.